app.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+(function(){
+  angular
+  .module("todo", [])
+  .controller("todo_controller", ToDoController);
+
+  function ToDoController() {
+    var vm = this;
+    vm.tasks = [
+      {
+        task: "Finish to-do list homework",
+      },
+      {
+        task: "Learn AngularJS",
+      },
+      {
+        task: "Make lunches for the week",
+      }
+    ];
+
+    vm.create = function() {
+      vm.tasks.push({task: vm.formToDoText, done: false});
+
+      vm.formToDoText = '';
+    }
+
+    vm.delete = function(task_index) {
+      console.log(vm.tasks)
+      console.log(task_index)
```

Comment:
Great job with this delete method! Nice to see that you were using `console.log`s as well, just remember to take them out after :)